### PR TITLE
ADD: Advanced Operating Systems from U.C.B.

### PR DESCRIPTION
**Pull request by breda**
This is a 2013 class, but a very good one from U. C. Berkeley.

### CODE REVIEW

**Comment by ericdouglas** [comment]
@breda thanks for your help but you should add your indication [here](https://github.com/open-source-society/computer-science/blob/master/extras/free-courses.md#operating-systems).

**Comment by breda** [comment]
@ericdouglas haha sorry I didn't see that! Added.

File: README.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ Courses | Duration | Effort
 Courses | Duration | Effort
 :-- | :--: | :--:
 [Operating Systems and System Programming](https://www.youtube.com/view_play_list?p=-XXv-cvA_iBDyz-ba4yDskqMDY6A1w_c)| 10 weeks | 2-3 hours/week
-[Advanced Operating Systems: Structures And Implementation](https://www.youtube.com/watch?v=ToySNfwFOyc&list=PL-XXv-cvA_iB_5Q8G8kW5idSwNmXypmQE)| 10 weeks | 2-3 hours/week
+[Advanced Operating Systems: Structures And Implementation](https://www.youtube.com/playlist?list=PL-XXv-cvA_iB_5Q8G8kW5idSwNmXypmQE)| 10 weeks | 2-3 hours/week
```
Comment (breda) [inline README.md]: @ericdouglas sure! I removed it :)

**Comment by ericdouglas** [comment]
@breda many thanks! :smile:

**Comment by breda** [comment]
You're welcome @ericdouglas 😅

**Pull request merged**